_posts/2024-01-30-non-fungible-treasures.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Beyond Tokens: Non-Fungible Treasures for Decentralized Worldbuilding'
+author: Vincent Charlebois
+date: 2024-01-30
+acknowledgement: 
+excerpt: "Introducing 'Boolean Art' as a tool for the exploration of the Carrier Bag Theory of Infrastructure"
```

Comment:
@writerly03 I've brought the changes over from the google doc, thanks to your great editing work.
This excerpt/ subtitle wasn't present in the doc and is the last thing I would like to have your opinion on.

> excerpt: "Introducing 'Boolean Art' as a tool for the exploration of the Carrier Bag Theory of Infrastructure"

I find it a bit too literal and descriptive as a subtitle. Minimally, I could see 'tool' being replaced by concept but I'm curious if you think it should be completely reworked.
"Introducing 'Boolean Art' as a ~~tool~~ concept for the exploration of the Carrier Bag Theory of Infrastructure"

If I would be reworking it, I would explore a subtitle like the following:
"Shaping Collaborative Futures: Art, Technology, and Collective Storytelling in the Blockchain Era"

Thanks again for your feedback. It should be ready to publish before the end of the week.